docker/launch-opflexserver.sh

```diff
@@ -26,4 +26,4 @@ if [ ! -f ${OPFLEXSERVER_CONF} ]; then
 EOF
 fi
 
-exec ${OPFLEXSERVER} --policy=${OPFLEXSERVER_POL} --grpc_conf=${OPFLEXSERVER_CONF} $@
+exec ${OPFLEXSERVER} --level=debug2 --policy=${OPFLEXSERVER_POL} --grpc_conf=${OPFLEXSERVER_CONF} $@
```

Comment:
is this intentional?

Reply:
yes. without this no logs are printed for opflex server and its not clear if its hung or processing anything.